website/blog/2022-07-27-getting-started-with-the-dbt-semantic-layer.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Getting Started With The dbt Semantic Layer"
+description: "There's been a lot of discussion around dbt Metrics and what they are. Callum Mccann clarifies what the Semantic Layer is, what is available today, and how one can already begin to use parts of it!"
```

Comment:
Think we can make this a little more aspirational? Maybe let them know there's an interactive piece "Heard about dbt Metrics and curious to give them a try? Callum McCann walks us through an example metrics repo and how it fits into the dbt Semantic Layer."

Reply:
Updated